websites/K/Kittensgame/presence.ts

```diff
@@ -0,0 +1,127 @@
+const presence = new Presence({
+		clientId: "1218931024592113744",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Advice = "https://i.imgur.com/f5RrPFn.png",
+	Logo = "https://i.imgur.com/ZnH9imj.png",
+	Loading = "https://i.imgur.com/9sInqXl.gif",
```

Comment:
@Bas950 Do .gif files still have trouble when uploaded?
```suggestion
	Loading = "https://i.imgur.com/9sInqXl\u002egif",
```

Reply:
I think so, I can manually upload it if needed.